container/options.go

```diff
@@ -38,25 +38,49 @@ func applyOptions(c *Container, opts ...Option) error {
 	return nil
 }
 
+// ensure all the container identifiers are either empty or unique.
+func validateIds(c *Container, seen map[string]bool) error {
+	if c.opts.id == "" {
+		return nil
+	} else if seen[c.opts.id] {
+		return fmt.Errorf("duplicate container ID %q", c.opts.id)
+	}
+	seen[c.opts.id] = true
+
+	return nil
+}
+
+// ensure all the container only have one split modifier.
+func validateSplits(c *Container) error {
+	if c.opts.splitFixed > -1 && c.opts.splitPercent != 50 {
```

Comment:
(optional) We might want to use DefaultSplitPercent here instead of the literal 50. That way if anyone ever decides to change the value - they don't forget to change it here too.

https://github.com/mum4k/termdash/blob/0e5242ca6f8386889a303c67b58a25f98aed75b6/container/options.go#L200

Optional, it is unlikely we will be changing the default behavior as that would be a breaking change for multiple users.

Reply:
Did the change and changed the literal -1 to DefaultSplitFixed too. :ok_hand: